README.md

```diff
@@ -1,36 +1,35 @@
-`wrhs` CLI for Warehouse.ai
-====
+# `wrhs`
 
-Flawless CLI management of build pipelines & asset deployments for your front-end apps powered by [Warehouse.ai].
-
-# Configuration
-`wrhs` configuration can be passed in via CLI flags (more information on this below), or by creating a configuration file at `~/.wrhs`.
-### Example `.wrhs` file:
-```
-{
-  "hosts": {
-    "wrhs": "warehouse.ai",
-    "status": "warehouse-status.ai"
-  },
-  "auth": {
-    "user": "username",
-    "pass": "password"
-  }
-}
-```
+[![Version npm](https://img.shields.io/npm/v/wrhs.svg?style=flat-square)](https://www.npmjs.com/package/wrhs)
+[![License](https://img.shields.io/npm/l/wrhs.svg?style=flat-square)](https://github.com/warehouseai/wrhs/blob/master/LICENSE)
+[![npm Downloads](https://img.shields.io/npm/dm/wrhs.svg?style=flat-square)](https://npmcharts.com/compare/wrhs?minimal=true)
+[![Build Status](https://travis-ci.org/warehouseai/wrhs.svg?branch=master)](https://travis-ci.org/warehouseai/wrhs)
+[![Dependencies](https://img.shields.io/david/warehouseai/wrhs.svg?style=flat-square)](https://github.com/warehouseai/wrhs/blob/master/package.json)
 
-
-[![License](https://img.shields.io/npm/l/wrhs.svg)](https://github.com/warehouseai/wrhs/blob/master/package.json)
+CLI for [Warehouse.ai]. CLI to manage build pipelines & asset deployments
+for your front-end apps powered by [Warehouse.ai]. A useful addition to
+the [Warehouse.ai UI][wrhs-ui].
```

Comment:
>  A useful addition to the [Warehouse.ai UI][wrhs-ui].

I see the connection, but maybe reword as something more like `There is also a [web-based UI][wrhs-ui] available for [Warehouse.ai].`?